### ISSUE TITLE
Javscript Sandbox: Safe Mode

### ISSUE THREAD

**Issue by helloanoop** [usebruno/bruno#2807]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Bruno supports execution of JavaScript code in Variables, Scripts, Tests, and Assertions.
Today, we use [vm2](https://www.npmjs.com/package/vm2) to execute javascript.

This approach however allows the code execute system commands, access the file system and this is a security risk.

Most collections do not need to execute system commands or access the file system.

We are introducing `Safe Mode` to execute JavaScript code in a secure sandbox environment using [isolated-vm](https://github.com/laverdet/isolated-vm)
The user can switch to `Developer Mode` if the collection needs access to the file system or system commands, provided the user is aware of the security risks and trusts the source of the collection.

### Mockups or Images of the feature

<img width="1102" alt="image" src="https://github.com/user-attachments/assets/bcc7a7bd-e973-4ff4-b5f1-6bfeaca4e17e">

**Comment by helloanoop** [comment]
The PR is almost ready. We are doing some testing before getting to merged to main and moving forward with a release.

**Comment by mkurapov** [comment]
@helloanoop is there a way where we can make Safe Mode to be disabled via a config flag? We use scripts always by default in our dev environment (that import libraries), and want to make sure any new developers don't run into issues related to this.

**Comment by vikingtoby** [comment]
Any way to make it default to safe mode without asking whenever I create a new collection?

**Comment by nathanchere** [comment]
Making this a nag prompt on EVERY SINGLE SCRIPT OPENED is insane.

**Comment by anusreesubash** [comment]
Yes. We will consider this. An option to set either safe or developer mode as default make sense.
